realsense2_camera/launch/rs_launch.py

```diff
@@ -27,56 +27,57 @@
-                           {'name': 'pointcloud.stream_filter',     'default': '2', 'description': 'texture stream for pointcloud'},
-                           {'name': 'pointcloud.stream_index_filter','default': '0', 'description': 'texture stream index for pointcloud'},
-                           {'name': 'pointcloud.ordered_pc',        'default': 'false', 'description': ''},
-                           {'name': 'enable_sync',                  'default': 'false', 'description': "'enable sync mode'"},
-                           {'name': 'enable_rgbd',                  'default': 'false', 'description': "'enable rgbd topic'"},
-                           {'name': 'align_depth.enable',           'default': 'false', 'description': "'enable align depth filter'"},
-                           {'name': 'colorizer.enable',             'default': 'false', 'description': "''"},
                            {'name': 'clip_distance',                'default': '-2.', 'description': "''"},
+                           {'name': 'angular_velocity_cov',         'default': '0.01', 'description': "''"},
                            {'name': 'linear_accel_cov',             'default': '0.01', 'description': "''"},
-                           {'name': 'initial_reset',                'default': 'false', 'description': "''"},
-                           {'name': 'allow_no_texture_points',      'default': 'false', 'description': "''"},
+                           {'name': 'diagnostics_period',           'default': '0.0', 'description': 'Rate of publishing diagnostics. 0=Disabled'},
                            {'name': 'publish_tf',                   'default': 'true', 'description': '[bool] enable/disable publishing static & dynamic TF'},
                            {'name': 'tf_publish_rate',              'default': '0.0', 'description': '[double] rate in Hz for publishing dynamic TF'},
-                           {'name': 'diagnostics_period',           'default': '0.0', 'description': 'Rate of publishing diagnostics. 0=Disabled'},
-                           {'name': 'decimation_filter.enable',     'default': 'false', 'description': 'Rate of publishing static_tf'},
-                           {'name': 'rosbag_filename',              'default': "''", 'description': 'A realsense bagfile to run from as a device'},
-                           {'name': 'depth_module.exposure.1',      'default': '7500', 'description': 'Depth module first exposure value. Used for hdr_merge filter'},
-                           {'name': 'depth_module.gain.1',          'default': '16', 'description': 'Depth module first gain value. Used for hdr_merge filter'},
-                           {'name': 'depth_module.exposure.2',      'default': '1', 'description': 'Depth module second exposure value. Used for hdr_merge filter'},
-                           {'name': 'depth_module.gain.2',          'default': '16', 'description': 'Depth module second gain value. Used for hdr_merge filter'},
```

Comment:
Deleted:
- depth_module.exposure.1
- depth_module.gain.1
- depth_module.exposure.2
- depth_module.gain.2

Reply:
why deleted ?
We fetch these parameters in RosSensor::UpdateSequenceIdCallback

Reply:
We use these parameters just on initialization, and then delete them from the Node parameters (so, they don't appear at runtime)

Reply:
oh! yes, got it. will revert those changes.